detection/generate_transforms.py

```diff
@@ -39,12 +46,18 @@
     ConvertBoxModed,
     StandardizeEmptyBoxd,
 )
+from monai.config import KeysCollection
+from monai.utils.type_conversion import convert_data_type
+from monai.data.box_utils import clip_boxes_to_image
+from monai.apps.detection.transforms.box_ops import convert_box_to_mask
+from monai.transforms.utils_pytorch_numpy_unification import stack, concatenate, min, max
 
 
 def generate_detection_train_transform(
     image_key,
     box_key,
     label_key,
+    point_key,
```

Comment:
Hi @KumoLiu , would it make sense if we give a default value (`points`) to the `point_key`? Some users may not be interested to understand  what are the names of the points.

Reply:
Yes! Add a default value for point_key.